nbs/07_quickstart_for_web_devs.ipynb

```diff
@@ -191,29 +193,26 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Variable in URLs\n",
+    "## Variables in URLs\n",
     "\n",
-    "You can add variable sections to a URL by marking sections with `{variable_name}`. Your function then receives the `{variable_name}` as a keyword argument, but only if it is the correct type. Here's an example:\n",
+    "You can add variable sections to a URL by marking them with `{variable_name}`. Your function then receives the `{variable_name}` as a keyword argument, but only if it is the correct type. Here's an example:\n",
     "\n",
     "```{.python filename=\"main.py\" code-line-numbers=\"true\"}\n",
     "from fasthtml.fastapp import * \n",
     "\n",
     "app, rt = fast_app()\n",
     "\n",
-    "@rt(\"/\")\n",
```

Comment:
I removed the root `/` route because it felt redundant with the previous code example, but add it back in if it improves clarity.